bugbug/repository.py

```diff
@@ -378,6 +465,9 @@ def _transform(commit):
 
             if size is not None:
                 source_code_sizes.append(size)
+                metrics = code_analysis_server.metrics(path, after, unit=False)
```

Comment:
I forgot what the unit parameter is, could you document it in code_analysis_server?